pkg/controller/elasticsearch/client/model_test.go

```diff
@@ -161,3 +161,32 @@ func TestLicenseUpdateResponse_IsSuccess(t *testing.T) {
 		})
 	}
 }
+
+func TestShutdownResponse(t *testing.T) {
+	nodeShudownSample := `{"nodes":[{"node_id":"PQKHA4xCQd2xErO2fUK-hg","type":"REMOVE","reason":"2331481","shutdown_startedmillis":1643648932189,"status":"IN_PROGRESS","shard_migration":{"status":"IN_PROGRESS","shard_migrations_remaining":1},"persistent_tasks":{"status":"COMPLETE"},"plugins":{"status":"COMPLETE"}}]}`
```

Comment:
nit: to make it easier to read
```suggestion
	nodeShudownSample := `{
	"nodes": [{
		"node_id": "PQKHA4xCQd2xErO2fUK-hg",
		"type": "REMOVE",
		"reason": "2331481",
		"shutdown_startedmillis": 1643648932189,
		"status": "IN_PROGRESS",
		"shard_migration": {
			"status": "IN_PROGRESS",
			"shard_migrations_remaining": 1
		},
		"persistent_tasks": {
			"status": "COMPLETE"
		},
		"plugins": {
			"status": "COMPLETE"
		}
	}]
}`
```